src/tooltip/test/tooltip-template.spec.js

```diff
@@ -75,5 +75,22 @@ describe('tooltip template', function() {
     expect( elmBody.children().length ).toBe( 1 );
   }));
 
+  it('should open when templateUrl is specified as a string literal rather than an expression - issue 3852', inject(function($compile) {
+    elmBody = angular.element(
+      '<div><span tooltip-template="myUrl">Selector Text</span></div>'
```

Comment:
Maybe I need more coffee, but I'm having a little trouble understanding your comment.  The change I suggested is to support string literals - expressions are currently the only thing that is supported.  So, currently, something like `<span tooltip-template="myUrl">` doesn't work unless you wrap `myUrl` in single quotes as well.